.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
[Optional] you can add  `--format documentaion` for more verbose output

Reply:
We have it since day one but it does not work :-(